CONTRIBUTING.md

```diff
@@ -0,0 +1,90 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+Discord, or any other method with the owners of this repository before making a change.
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Pull Request Process
+
+1. Ensure any installation or build dependencies are removed before the end of the layer when doing a
+   build.
+2. Update the README.md (of specific package/app) with details of changes to the interface; this includes new environment
+   variables, exposed ports, useful file locations and container parameters.
+3. You may merge the Pull Request in once you have the sign-off of Core Team developers, or if you
+   do not have permission to do that, you may request the Core Team member to merge it for you.
+
+## Code of Conduct
+
+### Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to make participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
+nationality, personal appearance, race, religion, or sexual identity and
+orientation.
```

Comment:
In the interest of nurturing an open, inclusive, and progressive environment, we, as contributors, maintainers, and participants, commit to ensuring that participation in our project and our community is a respectful and harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.